docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -92,7 +92,7 @@ dockerOpts=" \
 $tnsMount \
 --mount type=bind,source=/etc/condor,target=/etc/condor,readonly \
 --mount type=bind,source=/tmp,target=/tmp \
---mount type=bind,source=$HOST_MOUNT_DIR/certs,target=/data/certs \
+--mount type=bind,source=$HOST_MOUNT_DIR/../certs,target=/data/certs \
```

Comment:
hi @amaltaro 
Provided we now know there is requirement for absolute match of the mount point and host path, I am more in favor of: 
```
--mount type=bind,source=/data/certs,target=/data/certs \
```